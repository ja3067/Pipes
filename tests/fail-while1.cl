def dynamic():
	if True:
		return 3
	else:
		return 4.0

print(dynamic())

while dynamic():
	print(3)
