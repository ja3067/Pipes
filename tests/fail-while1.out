3
RuntimeError: invalid boolean type in while statement
